Prog/Eviction Group/Pulse-housing-analysis.R

```diff
@@ -0,0 +1,141 @@
+######
+# Program: DC housing analysis PUFS Census Pulse Cleaning
+# Author: Elizabeth Burton 
+# Created: 1/17/24
+# Description: compile, clean, and analyze PUF census data for DC housing assistance analysis
+######
+
+library(tidyverse)
+library(readxl)
+library(openxlsx)
+library(data.table)
+library(srvyr)
+library(weights)
+
+# Read in data and replicate weights
```

Comment:
Current code: `list.files()` extracts the names of all .csv files at the specified path. `map_dfr()` and `read_csv` read in each .csv file and combine them all into a single dataset, called `data`. Resulting dataset has 629,582 rows and 283 columns.

Note that you could also do this using base R (`lapply()`) and `dplyr` (`bind_rows()`), but `purrr::map_dfr` is more efficient because you only have to individually read through the data once.